### PR TITLE
Properly ignore folders at root of project

### PR DESCRIPTION
**Pull request by MasterOdin**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,8 +22,8 @@ composer.lock
 tests/log
 
 # ignore core development artifacts
-./db
-./migrations
+/db
+/migrations
```
Comment (dereuromark) [inline .gitignore:26]: I prefer a trailing slash to clarify folders vs files :)
But this is also fine for now

Comment (MasterOdin) [reply .gitignore]: Fixed, and added them to a handful of others, though I think I missed a handful of other folders.

**Comment by dereuromark** [comment]
I think I broke a test with my last merge

```
There was 1 failure:

1) Test\Phinx\Db\Adapter\MysqlAdapterTest::testsHasForeignKeyWithSchemaDotTableName
Failed asserting that false is true.
```

**Comment by MasterOdin** [comment]
Looks like Travis is passing, not sure what might be causing that test failure.

**Comment by dereuromark** [comment]
Maybe my local DB or mysql version.

**Pull request merged**